turbo.json

```diff
@@ -1,5 +1,5 @@
 {
-  "$schema": "https://turbo.build/schema.v1.json",
+  "$schema": "https://turbo.build/schema.v2.json",
```

Comment:
Should have been changed in previous PR that upgraded from v1 to v2 or turborepo. Does not cause issues, as far as I could tell, but should match what we are using.